.github/workflows/build.yml

```diff
@@ -174,7 +177,7 @@ jobs:
           cache-targets: false
       - run: cargo install cargo-semver-checks --locked
       - name: Check semver
-        run: cargo +stable semver-checks check-release
+        run: cargo +stable semver-checks check-release --default-features --features  const_fn,asm_const,step_trait,doc_auto_cfg
```

Comment:
Why was this changed? Can we revert this?

Reply:
If we don't add this, cargo semver tests the crate with the new feature enabled, and therefore still complains about missing `const`.

I agree this is not ideal, and it should be removed when the minor version is increased, however I'm not sure we can do without it...

Reply:
Can we only use `--default-features`?

Reply:
We can yes, I just wanted to include everything but my stuff to make the change minimal

Reply:
Fair enough, but let's go with just `--default-features`, some of the additional features are deprecated and the rest are covered by the default feature, so I don't think we need to add anything.